gdnative-bindings/build.rs

```diff
@@ -2,54 +2,41 @@ use gdnative_bindings_generator::*;
 
 use std::env;
 use std::fs::File;
-use std::io::BufWriter;
+use std::io::{BufWriter, Write as _};
 use std::path::PathBuf;
 use std::process::Command;
 
 fn main() {
     let out_path = PathBuf::from(env::var("OUT_DIR").unwrap());
 
-    let bindings_types_rs = out_path.join("bindings_types.rs");
-    let bindings_traits_rs = out_path.join("bindings_traits.rs");
-    let bindings_methods_rs = out_path.join("bindings_methods.rs");
+    let output_rs = out_path.join("generated.rs");
 
-    let mut types_output = BufWriter::new(File::create(&bindings_types_rs).unwrap());
-    let mut traits_output = BufWriter::new(File::create(&bindings_traits_rs).unwrap());
-    let mut methods_output = BufWriter::new(File::create(&bindings_methods_rs).unwrap());
+    {
+        let mut output = BufWriter::new(File::create(&output_rs).unwrap());
 
-    // gdnative-core already implements all dependencies of Object
-    let to_ignore = { strongly_connected_components(&Api::new(), "Object", None) };
+        // gdnative-core already implements all dependencies of Object
+        let to_ignore = { strongly_connected_components(&Api::new(), "Object", None) };
```

Comment:
Maybe remove the block here?

Reply:
Thank you! It had been using the old files locally and I missed it.

Reply:
I didn't see that! @karroffel had to point it out to me. :-/